app/assets/javascripts/search.js

```diff
@@ -0,0 +1,20 @@
+var $main        = $('#home_query'),
+    $name        = $('input#name'),
+    $summary     = $('input#summary');
```

Comment:
Use `var $summary = $('input#summary');` or `,` on the end of this (and following lines).